src/Reactduino.h

```diff
@@ -44,9 +50,13 @@ class Reactduino
     reaction repeat(uint32_t t, react_callback cb);
     reaction onAvailable(Stream *stream, react_callback cb);
     reaction onInterrupt(uint8_t number, react_callback cb, int mode);
+    reaction onInputChange(uint8_t number, react_callback cb, int state);
```

Comment:
Could we move this to being a private method?

As it's not documented and shouldn't really be used directory I feel it's not part of the public API.